tests/unittest/auto_deploy/singlegpu/shim/test_cached_sequence_interface.py

```
@@ -1063,3 +1118,206 @@ def dummy_host_prepare(batch_info_host: torch.Tensor, cu_num_pages_host: torch.T
+        device="cuda",
+        kv_cache_config=KvCacheConfig(
+            tokens_per_block=32, max_tokens=1024, free_gpu_memory_fraction=0.0
+        ),
+    )
+    interface.add_resource("kv_0", KVPagedResourceHandler(8, 64, dtype=torch.float16))
+    interface.add_resource("kv_1", KVPagedResourceHandler(4, 128, dtype=torch.float16))
+
+    interface.initialize_resources()
+
+    # Group 0 (head_dim=64): cache shape [..., 8, 32, 64]
+    kv_0 = interface._caches[list(interface._caches.keys())[0]]
```

Ruff (0.15.9)

[warning] Prefer `next(iter(interface._caches.keys()))` over single element slice

Replace with `next(iter(interface._caches.keys()))`

(RUF015)